### PR TITLE
Add new JVC Projector integration

### PR DESCRIPTION
**Pull request by SteveEasley**
## Proposed change

Adds documentation for new JVC Projector integration .

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/84748
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/3980
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/jvc_projector.markdown [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: JVC Projector
+description: Instructions on how to integrate JVC Projector into Home Assistant.
+ha_category:
+  - Remote
+ha_release: '2023.1'
```
Comment (Kane610) [inline source/_integrations/jvc_projector.markdown]:
```suggestion
ha_release: '2023.2'
```
Too late for that :)

**Review by Kane610 — CHANGES_REQUESTED** [review]
Looks good!

**Review by Kane610 — DISMISSED** [review]

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by SteveEasley** [comment]
@frenck This integration is done for this release. I will add the additional mentioned features in the next release when I will have more time.

**Review by elupus — COMMENTED** [review]
Looks good, not sure what frenck meant with the ha_release changes .

File: source/_integrations/jvc_projector.markdown [hunk 2]
```diff
@@ -0,0 +1,46 @@
+---
+title: JVC Projector
+description: Instructions on how to integrate JVC Projector into Home Assistant.
+ha_category:
+  - Remote
+ha_release: '2023.2'
```
Comment (frenck) [inline source/_integrations/jvc_projector.markdown]:
```suggestion
ha_release: '2023.6'
```

**Comment by frenck** [comment]
> Looks good, not sure what frenck meant with the ha_release changes .

I've just added the change. The `ha_release` version in the frontmatter of the integration page indicates the version of Home Assistant this integration was introduced. It needs to match the version of when the parent PR was merged.

../Frenck

**Review by frenck — APPROVED** [review]

**Pull request merged**